src/features/selected-profile-request-headers/pasteMultiple/model.ts

```diff
@@ -0,0 +1,73 @@
+import { attach, createEvent, sample } from 'effector';
+
+import { $requestProfiles, $selectedRequestProfile, profileUpdated } from '#entities/request-profile/model';
+import { RequestHeader } from '#entities/request-profile/types';
+import { generateId } from '#shared/utils/generateId';
+
+import { DELIMITER, NEW_ROW } from '../paste/constant';
+
+type SelectedProfileRequestHeadersPasted = {
+  id: RequestHeader['id'];
+  value: string;
+  field: 'name' | 'value';
+};
+
+type ProcessValue = {
+  pastedValue: string;
+  header?: RequestHeader;
+};
+
+const formatValue = ({ pastedValue, header }: ProcessValue): { name: string; value: string } => {
+  const transformNameValue = pastedValue.split(DELIMITER).map(element => element.trim());
```

Comment:
Может сразу именованные поля брать, чтобы дальше не обращаться по индексам?

```suggestion
  const [name, value] = pastedValue.split(DELIMITER).map(element => element.trim());
```